docs/application/dotnet/tutorials/certificates/permit-device-to-install-apps.md

```diff
@@ -1,15 +1,16 @@
 # Permit Device To Install Applications
 
-In the menu bar, click **Tools** > **Tizen** > **Tizen Device Manager**.
+To permit a device to install applications, follow these steps:
 
-Right-click anywhere in the file explorer of the **Device Manager** and select **Permit to install applications**.
+1. In the Visual Studio menu, select **Tools** > **Tizen** > **Tizen Device Manager**.
+2. In the **Device Manager** window that appears, select the device and right-click. 
+3. Select **Permit to install applications** from the options.
 
-![Device Manager](media/DeviceManager.png)
+    ![Device Manager](media/DeviceManager.png)
```

Comment:
Request you to provide an updated screenshot capturing the latest version.

Reply:
We believe there is no major difference, but we will include the latest image.

We cannot push commits to a public repository inside the company network, instead we will provide a url for the image.

https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/permit-device-to-install-apps-2.png

Reply:
Thank you.